railsapp/app/models/recent.rb

```diff
@@ -1,3 +1,5 @@
 class Recent < ApplicationRecord
+
   belongs_to :product
+  belongs_to :user
```

Comment:
It seems that the change is already in master branch. Why did it appear again..??